### ISSUE TITLE
Image Import with nested arrays not working 

### ISSUE THREAD

**Issue by aterjung** [avstudnitz/AvS_FastSimpleImport#219]
Hi,

if i import category assignments with nested arrays like:
 '_category'=> array('test1','test2')  
or with ids
 '_category'=> array(1,2)  

i'll get a "Category does not exists" error. If i split the import into to rows, no error is thrown and the import is successful.

Is there anything known about this?

thanks in advance

Aike Terjung

**Comment by Nesse** [comment]
I'm having the same problem. I'm using the develop branch together with HO_Import.

**Comment by paales** [comment]
@Nesse Do not use nested arrays with Ho_Import, Ho_Import handles all the file formatting.

**Comment by Nesse** [comment]
@paales So it's not necessary to split my categories?

"Voetbal/Shirts lange mouw***Sportkleding & accessoires/Kleding/Shirts/Lange mouwen"

``` xml
<_category helper="ho_import/import::getFieldSplit">
     <field field="categories" />
      <split>***</split>
</_category>
``` 
![image](https://cloud.githubusercontent.com/assets/2901978/7589355/6cde1814-f8c2-11e4-8aa0-cf2f722580b2.png)

**Comment by paales** [comment]
Yes that is necessary, but you shouldn't use the nested arrays configuration is AvS_FastSimpleImport. Lets continue the discussion on the Ho_Import Github https://github.com/ho-nl/Ho_Import/issues/59
